src/store/metadata/oxigraph/store.py

```diff
@@ -252,6 +284,16 @@ def get_version(
         version_graph["table_schema"]["columns"] = columns_graph
         del version_graph["table_schema"]["@id"]
 
+        # if version_graph["issued"] is a list that contains ["2023-11-06T12:25:28", "2023-11-06T12:25:28+00:00"], 
```

Comment:
So some good figuring out here but I suspect this is a data error that has since been corrected (or its a data error local to your machine). If ever you hit unexpected data like this locally check the data on https://staging.idpd.uk/sparql and see if your local graph differs.

To clear our you local data:

- docker-compose down
- delete the `.data` folder

Either way we shouldn't be correcting incorrect data, making sure its correct is someone else's job.

At most check that this is a string as expected, if its _not_ then log an error so we know about it and return None.

Reply:
Just clicked this is what @SarahJohnsonONS was telling me about earlier, worth checking in with her and seeing if this is corrected on the data side.